eukaryote/constraints/semantics/sentence_encoders/universal_sentence_encoder/universal_sentence_encoder.py

```diff
@@ -19,7 +18,7 @@ def __init__(self, threshold=0.8, large=False, metric="angular", **kwargs):
         if large:
             tfhub_url = "https://tfhub.dev/google/universal-sentence-encoder-large/5"
         else:
-            tfhub_url = "https://tfhub.dev/google/universal-sentence-encoder/3"
+            tfhub_url = "https://tfhub.dev/google/universal-sentence-encoder/4"
```

Comment:
I'm guessing this was just something that's out of date in the original TA - should we be worried about this thing getting updated and breaking our version of TA eventually?